modules/side-panel/react/lib/SidePanel.tsx

```diff
@@ -166,13 +168,13 @@ export default class SidePanel extends React.Component<SidePanelProps, SidePanel
       backgroundColor,
       openNavigationLabel,
       closeNavigationLabel,
+      role,
       ...elemProps
     } = this.props;
 
     return (
       <SidePanelContainer
-        role="region"
-        aria-orientation="vertical"
+        role={role}
```

Comment:
Shouldn't this have been possible before because we extend `React.HTMLAttributes<HTMLDivElement>`?

It would have been spread with `{...elemProps}` and if we end up with 
```tsx
<SidePanelContainer
  role="region"
  // ...
  role={this.props.role}
>
```
Where the second role prop would override the first

Reply:
oh herm.. I think either way we shouldn't enforce the aria attributes, and these two together doesn't pass Lighthouse as per the issue. Should we still change it?

Reply:
So the problem was more around having `role="region"` and `aria-orientation="vertical"` by default?

Reply:
correct! @anicholls

Reply:
So why not just remove `aria-orientation="vertical"`? `role="region"` is still the correct default, right? Consumers still have the ability to override it without explicitly setting it as a prop. I just question it because we don't have the `role` exposed in any other component interfaces.

Reply:
the drawer component in Labs has a role as well but no default, so I was going off of what I did there for this use case. I'm ok with adding it back in and setting it as a default!